### PR TITLE
Add changes to manage secrets

### PR DESCRIPTION
**Pull request by feyukio**
Add changes to manage secrets variables.

### CODE REVIEW

File: install_dependencies.sh [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 apk update
 
 # Install dependencies
-apk add --no-cache --virtual build-dependencies gcc python3-dev musl-dev
-apk add postgresql-dev make
+apk add --no-cache --virtual build-dependencies \
+    gcc \
+    python3-dev \
+    musl-dev \
+    make
```
Comment (Patriciakikunaga) [inline install_dependencies.sh]: Why do you need this `make`?

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+ENV DOCKERIZE_VERSION=v0.6.1
+
+# Install gcredstash and dockerize
+RUN set -ex \
+    && apk update && apk add wget ca-certificates \
+    && wget -O - https://github.com/winebarrel/gcredstash/releases/download/$CREDSTASH_VERSION/gcredstash-$CREDSTASH_VERSION-linux-amd64.gz | gunzip -c  > /usr/local/bin/gcredstash \
+    && wget -O - https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz | tar -C /usr/local/bin -xzv \
+    && chmod -v 755 /usr/local/bin/gcredstash /usr/local/bin/dockerize
+
+COPY requirements.txt install_dependencies.sh entrypoint.sh /
```
Comment (Patriciakikunaga) [inline Dockerfile]: Do you need to copy the file `entrypoint.sh`? It is already here.

File: entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
```
Comment (Patriciakikunaga) [inline entrypoint.sh]: Why this file is here and not in the watchdog repository?
You are not using it.

File: Dockerfile (hunk 2 above)
Comment (feyukio) [reply Dockerfile]: The files only exists if they are copy.

File: install_dependencies.sh (hunk 1 above)
Comment (feyukio) [reply install_dependencies.sh]: The sanic has uvloop as dependency and to build uvloop, the make is needed.

File: entrypoint.sh (hunk 3 above)
Comment (feyukio) [reply entrypoint.sh]: I am not sure if this file can be open when Watchdog become open source.

**Pull request merged**